src/main/charts/agent/values.yaml

```diff
@@ -0,0 +1,233 @@
+  #
+  ports:
+  
+    # -- The port on which the Bamboo agent listens for HTTP traffic
+    #
+    http: 80
+     
+  # TODO: As of yet a /status endpoint does not exist for agents. Once delivered the implementation for this
+  # must be added
+  # Confirm that the Bamboo agent is up and running with a ReadinessProbe
+  # https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes
+  #
+  readinessProbe:
+  
+    # -- The initial delay (in seconds) for the Bamboo agent container readiness probe, 
+    # after which the probe will start running.
+    #
+    initialDelaySeconds: 10
+    
+    # -- How often (in seconds) the Bamboo agent container readiness probe will run
+    #
+    periodSeconds: 5
+    
+    # -- The number of consecutive failures of the Bamboo agent container readiness probe 
+    # before the pod fails readiness checks.
+    #
+    failureThreshold: 30
+
+  shutdown:
+
+    # -- The termination grace period for pods during shutdown. This
+    # should be set to the internal grace period, plus a small buffer
+    # to allow the JVM to fully terminate.
+    #
+    terminationGracePeriodSeconds: 30
+
+    # -- By default pods will be stopped via a [preStop hook](https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/),
+    # using a script supplied by the Docker image. If any other
+    # shutdown behaviour is needed it can be achieved by overriding
+    # this value. Note that the shutdown command needs to wait for the
+    # application shutdown completely before exiting; see [the default
+    # TODO: This needs to be updated when Steve's changes are done
+    # command](https://bitbucket.org/atlassian-docker/docker-atlassian-jira/src/master/shutdown-wait.sh)
+    # for details.
+    #
+    command: "/shutdown-wait.sh"
+
+  # Pod resource requests 
+  #
+  resources:
+  
+    # JVM Memory / Heap Size definitions. These values below are based on the 
+    # defaults defined for the Bamboo agent docker container.
+    # https://bitbucket.org/atlassian-docker/docker-bamboo-agent-base
+    #
+    jvm:
+      
+      # -- The maximum amount of heap memory that will be used by the Bamboo agent JVM
+      #
+      maxHeap: "512m"
+      
+      # -- The minimum amount of heap memory that will be used by the Bamboo agent JVM
+      #
+      minHeap: "256m"
+      
+    # Specifies the standard K8s resource requests and/or limits for the Bamboo agent
+    # container. It is important that if the memory resources are specified here, 
+    # they must allow for the size of the Bamboo agent JVM. That means the maximum heap 
+    # size, plus other JVM overheads, must be accommodated. 
+    # Allowing for (maxHeap)*1.5 would be an example.
+    #
+    container:
+      
+      requests:
+      
+        # -- Initial CPU request by Bamboo agent pod
+        #
+        cpu: "2" # If changing the cpu value update 'ActiveProcessorCount' below
```

Comment:
Are those recommended values for Bamboo agent?

Reply:
There are only [recommendations for memory requirements](https://bitbucket.org/dchevell/docker-atlassian-bamboo-base-agent/src/master/) but not CPU. I figured a CPU count of `2` seemed reasonable but maybe not?

Reply:
2 vcpu is more than enough for the agent itself but there's certainly no standards here, especially when it comes to what kind of jobs the agent will take. In fact, 2 vcpu can be a lot fo ask (as a default), provided that one has 4cpu/16GB workers for example.

Reply:
@bianchi2 2 does feel a bit excessive as a default, I think we can change this to `1` and leave it up to the user of the chart to tune this as they see fit.